project.yaml

```diff
@@ -1382,10 +1392,21 @@ Layer:
             way,
             place,
             name,
-            ref
+            ref,
+            CASE
+              WHEN (population ~ '^[0-9]+$') THEN CAST(population AS integer)
```

Comment:
Is there a difference between `CAST(population AS integer)` and `population::integer`?